### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by AmalieDue**
### Description
Updated the Encryptonize Core README. How detailed should the Repository overview section be? Or should it be removed?

### Relevant Issues/PRs
Closes DEV-133

### Type(s) of Change (Split the PR if you check many)
- [ ] Added user feature
- [ ] Added technical feature
- [ ] Added tests
- [ ] Refactor
- [ ] Bugfix
- [x] Updated documentation
- [ ] Kubernetes changes
- [ ] CI/CD workflow changes

### Testing Checklist
- [ ] I have added/updated unit tests for functional changes.
- [ ] I have added/updated integration tests for changes that affect dependent systems.
- [ ] I have added/updated end-to-end tests for feature changes.

### General Checklist
- [x] I have pulled in the latest changes from master.
- [ ] I have run `make lint`.
- [ ] I have successfully run `make tests`.
- [ ] I have updated relevant CI/CD workflows.
- [ ] I have updated relevant Kubernetes manifests/scripts.
- [x] I have updated/added relevant (internal and external) documentation (readme, doc comments, etc).
- [ ] I have updated the dependency map to reflect my changes.
- [ ] I have added the license to new source code files.
- [ ] I have spent some time looking over the full diff before creating this PR.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
-This will start a complete Encryptonize&reg; instance inside docker. To shut down the
-Encryptonize&reg; instance, call
+This will start a complete Encryptonize Core instance inside Docker. To shut down the
+Encryptonize Core instance, call
 
 ```bash
 make docker-down
 ```
 
+For more information about how to get started with Encryptonize Core, see [documentation/getting_started.md](documentation/getting_started.md). The documentation folder also provides a [documentation/user_manual.md](documentation/user_manual.md) as well as an overview of the [documentation/api.md](documentation/api.md).
```
Comment (philip-cc) [inline README.md:31]:
```suggestion
For more information about how to get started with Encryptonize Core, see the [Getting Started guide](documentation/getting_started.md). The documentation folder also provides a [User Manual](documentation/user_manual.md) as well as an overview of the [gRPC API](documentation/api.md).
```

File: README.md [hunk 2]
```diff
@@ -41,6 +42,42 @@ See the [provisioning README](deployment/provision/README.md) for more details.
...
+## Running Encryptonize&reg; Core manually
 
-See the Encryptonize&reg; Core Helm Chart [README](deployment/charts/encryptonize-core/README.md) for more information.
+Encryptonize Core can also be run manually using the gRPC API. Note that this is only recommended for development. To do this, build the executable:
+
+```bash
+./encryptonize-core
+```
+
+Afterwards, the gRPC service endpoints can be called with e.g. the tool [gRPCurl](https://github.com/fullstorydev/grpcurl). See the [documentation/user_manual.md](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md) for more information.
```
Comment (philip-cc) [inline README.md:57]:
```suggestion
Afterwards, the gRPC service endpoints can be called with e.g. the tool [gRPCurl](https://github.com/fullstorydev/grpcurl). See the [User Manual](https://github.com/cyber-crypt-com/encryptonize-core/blob/master/documentation/user_manual.md) for more information.
```

File: README.md [hunk 3]
```diff
@@ -41,6 +42,42 @@ See the [provisioning README](deployment/provision/README.md) for more details.
...
+
+See the Encryptonize Core Helm Chart [README](deployment/charts/README.md) for more information.
+
+## Running Encryptonize&reg; Core manually
 
-See the Encryptonize&reg; Core Helm Chart [README](deployment/charts/encryptonize-core/README.md) for more information.
+Encryptonize Core can also be run manually using the gRPC API. Note that this is only recommended for development. To do this, build the executable:
+
+```bash
+./encryptonize-core
```
[the remark this answers is not in the document]
Comment (AmalieDue) [reply README.md:54]: Fixed

File: README.md [hunk 4]
```diff
@@ -41,6 +42,43 @@ See the [provisioning README](deployment/provision/README.md) for more details.
...
 
-Using the Encryptonize&reg; Core Helm Chart, you can deploy Encryptonize&reg; Core to a Kubernetes.
+Using the Encryptonize Core Helm Chart, you can deploy Encryptonize Core to a Kubernetes.
+
+See the Encryptonize Core Helm Chart [README](deployment/charts/README.md) for more information.
+
+## Running Encryptonize&reg; Core manually
 
-See the Encryptonize&reg; Core Helm Chart [README](deployment/charts/encryptonize-core/README.md) for more information.
+Encryptonize Core can also be run manually using the gRPC API. Note that this is only recommended for development. To do this, run the executable:
```
Comment (philip-cc) [inline README.md:51]:
```suggestion
Encryptonize Core can also be run manually. Note that this is only recommended for development. To do this, run the executable:
```

File: README.md [hunk 5]
```diff
@@ -41,6 +42,43 @@ See the [provisioning README](deployment/provision/README.md) for more details.
...
+├── storage                     # Storage files
+├── .dockerignore
+├── Dockerfile                  
+├── README.md                   # This file
+├── config.toml                 # Example configuration file
+├── docker-compose.yml          
+├── go.mod
+├── go.sum
+├── main.go
+└── makefile
```
Comment (philip-cc) [inline README.md:83]: I think we should just explain the folders that are relevant, e.g.:
```suggestion
├── deployment                  # Deployment information
├── documentation               # User documentation
│   ├── api.md                      # API specifications
│   ├── getting_started.md          # Getting started guide
│   └── user_manual.md              # User manual
├── e2etests                    # End-to-end tests
├── protobuf                    # Protobuf files
├── service                     # Source files for Encryptonize Core
├── Dockerfile                  # Encryptonize Core Docker image definition
├── README.md                   # This file
├── config.toml                 # Example configuration file
├── docker-compose.yml          # Example Docker Compose setup
```

**Review by philip-cc — APPROVED** [review]

**Pull request merged**